pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md

```diff
@@ -43,32 +43,32 @@ If you are using a [private Docker registry][docker_registry] for your clusters,
 After you have the available Konvoy versions, you can upgrade your CLI by running the following command:
 
 ```bash
-konvoy image upgrade --version=v1.8.4
-Wrote Konvoy CLI version 'v1.8.4' to '.konvoy/cli_version'
+konvoy image upgrade --version=v1.8.5
+Wrote Konvoy CLI version 'v1.8.5' to '.konvoy/cli_version'
 ```
 
 After the upgrade command completes, you can start using the new Konvoy version.
 
-### Upgrading Konvoy from v1.7.x to v1.8.4
+### Upgrading Konvoy from v1.7.x to v1.8.5
 
 **You must modify your `cluster.yaml` with these changes when upgrading from a previous Konvoy version:**
 
 It is recommended to upgrade to the newest supported version of Kubernetes. Set `spec.kubernetes.version: 1.20.13` for of kind: ClusterConfiguration.
```

Comment:
I've never noticed this before, but the phrasing 'for of kind' in all these items is very strange and awkward.

```suggestion
It is recommended to upgrade to the newest supported version of Kubernetes. Set `spec.kubernetes.version: 1.20.13` for kind: ClusterConfiguration.
```